dts/bindings/dma/st,stm32-dma.yaml

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (c) 2019, Song Qiang <songqiang1304521@gmail.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+title: STM32 DMA
+
+description: >
+    This is a representation of the STM32 DMA nodes
```

Comment:
Not specific to this PR, but could use a shorter and more direct description like `description: STM32 DMA controller`.

The description shows up as a comment above any generated #defines.